### ISSUE TITLE
About the loss function

### ISSUE THREAD

**Issue by hongsiyu** [JunjH/Revisiting_Single_Depth_Estimation#22]
I use your loss function and your optimizer in my own network, but it doesn't work well. Can you tell me how to determine and adjust the learning rate when you train the model with your loss function?

**Comment by JunjH** [comment]
What does the training curve look like? I think the learning rate of 1e-4 is appropriate for adam. You may check if there are any issues in your code.

**Comment by hongsiyu** [comment]
> What does the training curve look like? I think the learning rate of 1e-4 is appropriate for adam. You may check if there are any issues in your code.

Thanks for your reply!I use your strategy to adjust the learning rate and the final rmse in the test set is higher than 1.0. When  the learning rate is 1e-3, the better rmse is approximately equal to 0.65. But I can get good result while I use Berhu loss. Do you know the reason?
